### DISCUSSION TITLE
How can I synchronize one database across multiple devices without relying on the central authority?

### DISCUSSION THREAD

**Discussion by salemalem** [simolus3/drift#1362] [Q&A]
How can I synchronize one on multiple computers on the socket level but without relying on the central figure?

**Comment by simolus3** [comment]
This is a very hard problem, and neither sqlite3 nor moor are trying to solve it. You might want to look at [Dqlite](https://dqlite.io/), [rqlite](https://github.com/rqlite/rqlite) or [Bedrock](https://bedrockdb.com/) for alternatives. It should be possible to integrate each of them into moor. You can use custom database engines with moor by importing `package:moor/backends.dart` and implementing `DatabaseDelegate`.
